### PR TITLE
ipalfish_tech_platform.md

### PR DESCRIPTION
**Pull request by JHF-skyman**
伴鱼数据质量平台实践文章提交

### CODE REVIEW

**Review by CalvinKirs — COMMENTED** [review]
Hi, thanks for your contribution, the content looks great. But it doesn't seem to have anything to do with Dolphin. It may not fit on the official website.

File: blog/zh-cn/ipalfish_tech_platform.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+
+规则实体是基于规则模板构建的，是规则的具象表达。在规则实体中将明确规则的 Expected Value、比较方式中具体的比较算子、参数的含义以及其他的一些元信息。基于同一个规则模板，可以构造多个规则实体。下图为「某表 user_id 唯一性校验」规则的示例：
+
+![Rule_entity](/img/ipalfish_platform/Rule_entity.png)
+
+值得一提的是，规则可能不仅仅只是针对单表的校验，对于多表的情况我们这套规则模板同样是适用的，只要我们可以将逻辑使用 SQL 表达。
+
+### 3. 规则绑定
+
+在 DS 的前端交互上支持为任务直接绑定校验规则，规则列表通过 API 从 DQC 获取，这种方式在用户的使用体验上存在一定的割裂（规则创建和绑定在两个平台完成）。同时，在 DQC 的前端亦可以直接设置关联调度，为已有任务绑定质检规则，任务列表通过 API 从 DS 获取。同一个任务可绑定多个质检规则，这些信息将存储至 DS 的 DAG 元信息中。那么这里需要考虑几个问题：
```
Comment (CalvinKirs) [inline blog/zh-cn/ipalfish_tech_platform.md:164]: It is better to change it to DolphinScheduler, not to use abbreviations.

Comment (JHF-skyman) [reply blog/zh-cn/ipalfish_tech_platform.md:164]: `### 背景补充

伴鱼离线调度开发平台是基于 Apache DolphinScheduler （简称 DS）实现的。`
A unified explanation has been given here. Is it still need to be modified below?

Comment (CalvinKirs) [reply blog/zh-cn/ipalfish_tech_platform.md:164]: Good job, Modify DS to DolphinScheduler. Don't use abbreviations.

Comment (JHF-skyman) [reply blog/zh-cn/ipalfish_tech_platform.md]: okay, I have submitted the question of modifying the abbreviations,  and rearranged the MD file according to the official documentation. Thanks for your advice

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Pull request merged**